lib/public_key/src/.gitignore

```diff
@@ -0,0 +1,2 @@
+OTP-PUB-KEY.*
+PKCS-FRAME.*
```

Comment:
Maybe something related to asn1 folder content is not needed anymore in `./lib/public_key/.gitignore` and can be removed?

Reply:
hmm, yes. will fix.

Reply:
I removed the new .gitignore and changed the rules in `lib/public_key/.gitignore` instead.